README.md

```diff
@@ -21,7 +21,11 @@ cerberus:
     distribution: openshift                              # Distribution can be kubernetes or openshift
     kubeconfig_path: ~/.kube/config                      # Path to kubeconfig
     watch_nodes: True                                    # Set to True for the cerberus to monitor the cluster nodes
-    watch_cluster_operators: True                        # Set to True for cerberus to monitor cluster operators. Parameter is optional, will set to True if not specified 
+    watch_cluster_operators: True                        # Set to True for cerberus to monitor cluster operators. Parameter is optional, will set to True if not specified
+    watch_url_routes:                                    # Route url's you want to monitor
```

Comment:
What routes should I have in the config by default? A user would need to change the url's so they are specific to their cluster, correct? And if they have need a bearer token they would have to login. Should I add something like the below where the user would have to fill in the necessary information to run properly?

watch_url_routes:
        - - https://prometheus-k8s-openshift-monitoring.apps.****.qe.devcluster.openshift.com
          - Bearer ****

Reply:
Maybe we can just add the watch_url_routes option and leave it empty since the url's are specific to user's cluster like you mentioned? Adding a brief comment about the usage should do it since it's explained in the readme I guess.

Reply:
I added the key and a comment about the format in the config, thoughts?

Reply:
Looks good, just the spacing for the comment needs to be fixed.